src/databricks/labs/remorph/cli.py

```diff
@@ -119,5 +119,16 @@ def configure_secrets(w: WorkspaceClient):
     recon_conf.prompt_and_save_connection_details()
 
 
+@remorph.command
+def reconcile_aggregate(w: WorkspaceClient):
+    """[EXPERIMENTAL] reconciles source to databricks datasets"""
```

Comment:
```suggestion
    """[EXPERIMENTAL] reconciles aggregates of source to databricks datasets"
    for example: COUNT, SUM between source and target with or without any specific key columns""
```

Reply:
Invocation will be addressed in a seperate PR